docs/content/reference/install-configuration/observability/healthcheck.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Traefik Health Check Documentation"
+description: "In Traefik Proxy, CLI & Ping lets you check the health of your Traefik instances. Read the technical documentation for configuration examples and options."
+---
+
+# CLI & Ping
+
+Checking the Health of your Traefik Instances
+{: .subtitle }
+
+## CLI
+
+The CLI can be used to make a request to the `/ping` endpoint to check the health of Traefik. Its exit status is `0` if Traefik is healthy and `1` otherwise.
+
+This can be used with [HEALTHCHECK](https://docs.docker.com/engine/reference/builder/#healthcheck) instruction or any other health check orchestration mechanism.
+
+### Usage 
+
+```sh
+traefik healthcheck [command] [flags] [arguments]
+```
+
+Example:
+
+```sh
+$ traefik healthcheck
+OK: http://:8082/ping
+```
+
+## Ping
+
+The `/ping` health-check URL is enabled with the command-line `--ping` or config file option `[ping]`.
+
+The `entryPoint` where the `/ping` is active can be customized with the `entryPoint` option,
```

Comment:
Same here do we need the backticks surrounding entrypoints in this page? and what about the way of writing the word?